eea/index.html

```diff
@@ -37,7 +37,7 @@ <h1 class="display-5 fw-bold">
       <div class="container">
         <div class="card mb-3">
           <div class="card-body">
-            <form action="" method="GET">
+            <form id="form">
```

Comment:
```suggestion
            <form id="form" novalidate>
```
Hier noch novalidate rein, damit es einheitlich ist, sonsts passts :+1: